.circleci/config.yml

```diff
@@ -41,7 +41,7 @@ commands:
     steps:
       - run:
           name: cargo build
-          command: cargo build
+          command: cargo build --release
```

Comment:
This shouldn't be necessary. We build syncstorage twice in CI (sigh): here and within the Docker. They don't share target/ directories (at one point they did, but grpc support code became out of sync between the environments, so we stopped  that).

The build profile in the Docker is what counts for our perf and it's already building in release mode (via cargo install which defaults to release)

Reply:
Ah, cool. 
I'll close this PR then.